exporters/metric/datadog/datadog.go

```diff
@@ -66,12 +66,11 @@ func defaultFormatter(namespace, name string) string {
 	return name
 }
 
-func (e *Exporter) Export(ctx context.Context, _ *resource.Resource, cs export.CheckpointSet) error {
-	// TODO: Use the Resource argument.
+func (e *Exporter) Export(ctx context.Context, cs export.CheckpointSet) error {
 	return cs.ForEach(func(r export.Record) error {
 		agg := r.Aggregator()
 		name := e.sanitizeMetricName(r.Descriptor().LibraryName(), r.Descriptor().Name())
-		itr := r.Labels().Iter()
+		itr := label.NewMergeIterator(r.Resource().LabelSet(), r.Labels())
```

Comment:
Oh, one thing: this call results in Resource labels overriding the metric labels, in case of conflict. I believe OTel recommends a policy of letting the metric label override the resource value, and likewise for Spans. I think we should reverse these arguments.